lib/features/support/app-automate.rb

```diff
@@ -0,0 +1,63 @@
+require 'appium_lib'
+require_relative './driver'
+
+class AppAutomateDriver < Driver
+
+  attr_reader :device_type
+
+  APP_UPLOAD_URI = "https://api-cloud.browserstack.com/app-automate/upload"
+
+  def devices
```

Comment:
Should this be public or private?

Reply:
I'm not too worried about methods being private, as it's a potentially useful method which may be used in the future by other feature sets.